### ISSUE TITLE
Add support for RDS cluster snapshot tags

### ISSUE THREAD

**Issue by atraver** [terraform-providers/terraform-provider-aws#8722]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

The `aws_db_cluster_snapshot` resource currently does not support the `tags` argument, although adding tags is supported via the REST API, CLI, and console.

### New or Affected Resource(s)

* aws_db_cluster_snapshot

### Potential Terraform Configuration

```hcl
resource "aws_db_cluster_snapshot" "test" {
  db_cluster_identifier = "some-identifier"
  db_cluster_snapshot_identifier = "my-test-snapshot"

  tags {
    "my:application" = "test"
  }
}
```

### References

* https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBClusterSnapshot.html#API_CreateDBClusterSnapshot_RequestParameters
* https://docs.aws.amazon.com/cli/latest/reference/rds/create-db-cluster-snapshot.html#options

**Comment by aybuke** [comment]
Hi @atraver, i want to working on this issue.

**Comment by adharmawan** [comment]
I'd like to work on this

**Comment by adharmawan** [comment]
@bflad could you take a look?

**Comment by bflad** [comment]
Support for retrieving tags in the `aws_db_cluster_snapshot` data source and managing tags in the `aws_db_cluster_snapshot` resource has been merged and will release with version 2.35.0 of the Terraform AWS Provider, Thursday next week. Thanks to @adharmawan for the implementation. 👍
